meson.build

```diff
@@ -79,9 +86,13 @@ if get_option('enable-avx')
   if get_option('platform') != 'android'
     if target == 'x86_64' or target == 'x86'
       extra_defines += '-DUSE_AVX=1'
-      add_project_arguments(['-march=native'], language: ['c','cpp'])
-      add_project_arguments(['-mavx2'], language: ['c','cpp'])
-      message('-march=native added for AVX hardware acceleration.')
+      if host_machine.system() == 'windows'
+        add_project_arguments(['/arch:AVX2'], language: ['c','cpp'])
```

Comment:
Also possible in native-file
cpp_args=...
cpp_link_args=...
we still need `if != `windows`